src/content/reference/react/useTransition.md

```diff
@@ -64,9 +64,30 @@ function TabContainer() {
 }
 ```
 
+It can also be used to wrap an async function to create an Action:
+
+```js {6}
+function TabButton({data, setData}) {
+  const [isPending, startTransition] = useTransition();
+
+  function updateAction(data) {
+    startTransition(async () => {
+      const newData = await updateData(data);
+      
+      // Note: currently, an additional startTransition
+      // is needed after any async requests. See Caveats.
+      startTransition(() => {
+        setData(data);
+      });
+    });
+  }
+  // ...
+}
+```
+
 #### Parameters {/*starttransition-parameters*/}
 
-* `scope`: A function that updates some state by calling one or more [`set` functions.](/reference/react/useState#setstate) React immediately calls `scope` with no parameters and marks all state updates scheduled synchronously during the `scope` function call as Transitions. They will be [non-blocking](#marking-a-state-update-as-a-non-blocking-transition) and [will not display unwanted loading indicators.](#preventing-unwanted-loading-indicators)
+* `scope`: A function that updates some state by calling one or more [`set` functions](/reference/react/useState#setstate). React immediately calls `scope` with no parameters and marks all state updates scheduled synchronously during the `scope` function call as Transitions. Any async calls awaited in the `scope` will be included in the transition, but currently require wrapping any `set` functions after the request in an additional `startTransition` (see [Troubleshooting](#react-doesnt-treat-my-state-update-after-await-as-a-transition)). State updates marked as Transitions will be [non-blocking](#marking-a-state-update-as-a-non-blocking-transition) and [will not display unwanted loading indicators.](#preventing-unwanted-loading-indicators).
```

Comment:
```suggestion
* `scope`: A function that updates some State by calling one or more [`set` functions](/reference/react/useState#setstate). React immediately calls `scope` with no parameters and marks all state updates scheduled synchronously during the `scope` function call as Transitions. Any async calls awaited in the `scope` will be included in the transition, but currently require wrapping any `set` functions after the request in an additional `startTransition` (see [Troubleshooting](#react-doesnt-treat-my-state-update-after-await-as-a-transition)). State updates marked as Transitions will be [non-blocking](#marking-a-state-update-as-a-non-blocking-transition) and [will not display unwanted loading indicators.](#preventing-unwanted-loading-indicators).
```

Reply:
I see a lot of state references from react to be in smallcase? Is this intentional?

Reply:
Yeah, i'll do a pass before merging. It's annoying to have every other word capitalized though